libs/commons/Common.ml

```diff
@@ -1219,6 +1219,16 @@ let files_of_dir_or_files_no_vcs_nofilter xs =
          else [ x ])
   |> flatten
 
+let chdir_to_repo_root () =
```

Comment:
This has nothing to do in Common.ml
libs/commons/ is for code independent of Semgrep, but the error message here is semgrep specific.
Moreover Common.ml is for functions that are very often used; we should limit to 50 functions at most.